### ISSUE TITLE
Elementor: Uncaught TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/so-widgets-bundle#547]
Elementor 1.7 ([changelog](https://wordpress.org/plugins/elementor/#developers)). was just released and upon opening the editor, the following error will appear in the console:

```
siteorigin-panels-2511.min.js?ver=2.5.11:formatted:1299 Uncaught TypeError: Cannot read property 'replace' of undefined
    at Object.processTemplate (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:1299)
    at Object.s.29 (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:3680)
    at l (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:17)
    at siteorigin-panels-2511.min.js?ver=2.5.11:formatted:19
    at Object.s.16../collection/cells (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:1376)
    at l (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:17)
    at e (siteorigin-panels-2511.min.js?ver=2.5.11:formatted:25)
    at siteorigin-panels-2511.min.js?ver=2.5.11:formatted:27
```

I suspect this may be causing #546

**Comment by braamgenis** [comment]
That's the SO Page Builder JS throwing that error. While I don't think we can fully support that as a use case, it would be nice if it didn't throw any errors. That said, I can't reproduce the issue, I have PB, WB and Elementor active and I don't see any JS errors.

**Comment by braamgenis** [comment]
Fixed in siteorigin/siteorigin-panels@e681d6e04b47f6b1c1c45a17f52f4a8064c91160
